prodserver

```diff
@@ -0,0 +1,24 @@
+#!/usr/local/bin/node
```

Comment:
can we use `#!/usr/bin/env node` because node may be installed in different locations across platforms
 [shebang](https://stackoverflow.com/questions/33509816/what-exactly-does-usr-bin-env-node-do-at-the-beginning-of-node-files)
`brew install node` you get 
![screen shot 2018-08-22 at 14 27 32](https://user-images.githubusercontent.com/22243090/44461215-a1df6c00-a618-11e8-9fd0-36e507f1ec45.png)
using `nvm` you get 
![screen shot 2018-08-22 at 14 25 52 1](https://user-images.githubusercontent.com/22243090/44461246-c0456780-a618-11e8-9d40-c745e75f429d.png)